common.ts

```diff
@@ -127,50 +127,10 @@ export const serializeDataIfNeeded = function (value: any, requestOptions: any,
         ? configuration.isJsonMime(requestOptions.headers['Content-Type'])
         : nonString;
     return needsSerialization
-        ? JSON.stringify(value !== undefined ? convertMapsAndSetsToPlain(value) : {})
+        ? JSON.stringify(value !== undefined ? value : {})
         : (value || "");
 }
 
-function convertMapsAndSetsToPlain(value: any): any {
```

Comment:
Why don't we need this anymore?

Reply:
I guess so, function has been dropped from generator [here](https://github.com/OpenAPITools/openapi-generator/compare/v7.5.0...v7.6.0) (look for modules/openapi-generator/src/main/resources/typescript-axios/common.mustache) and tests are green, so it seems that function is not needed anymore.